IdentityCore/src/MSIDError.m

```diff
@@ -99,10 +99,33 @@ MSIDErrorCode MSIDErrorCodeForOAuthError(NSString *oauthError, MSIDErrorCode def
     {
         return MSIDErrorServerAccessDenied;
     }
-    
+    if (oauthError && [oauthError caseInsensitiveCompare:@"tokenTransferFailedOTC"] == NSOrderedSame)
+    {   // Account Transfer session time out, When the token's time is expired
+        return MSIDErrorUserCancel;
+    }
     return defaultCode;
 }
 
+MSIDErrorCode MSIDErrorCodeForOAuthErrorWithSubErrorCode(NSString *oauthError, MSIDErrorCode defaultCode, NSString *subError)
+{
+    if (subError == nil) {
+        return MSIDErrorCodeForOAuthError(oauthError, defaultCode);
+    }
+//    if (oauthError && [oauthError caseInsensitiveCompare:@"invalid_grant"] == NSOrderedSame && [subError caseInsensitiveCompare:@"transfer_token_expired"] == NSOrderedSame )
+//    {   // When account Transfter Token is expired.
+//        return MSIDErrorUserCancel;
+//    }
+//    if (oauthError && [oauthError caseInsensitiveCompare:@"access_denied"] == NSOrderedSame && [subError caseInsensitiveCompare:@"tts_denied"] == NSOrderedSame )
+//    {   //when user cancels, this is the same error we return to mobile app for Account Transfer
+//        return MSIDErrorUserCancel;
+//    }
+//    if (oauthError && [oauthError caseInsensitiveCompare:@"access_denied"] == NSOrderedSame && [subError caseInsensitiveCompare:@"user_skipped"] == NSOrderedSame )
+//    {   //Account Transfter, when user skips the QR code page.
+//        return MSIDErrorUserCancel;
+//    }
```

Comment:
Are those commented out code expected?

Reply:
No, I was testing why is the build failing.